app/models/concerns/orchestration/dhcp.rb

```diff
@@ -89,7 +89,7 @@ def boot_server
   end
 
   def boot_server_or_proxy_hostname
-    return tftp.bootServer if tftp.bootServer.present?
+    return tftp.boot_server if tftp.boot_server.present?
```

Comment:
we could be more efficient while already touching this:
```suggestion
    return tftp.boot_server.presence
```

Reply:
No, that would be different. This is an early return with an if. Your suggestion would make the (deprecated) code below unreachable. We haven't made a formal deprecation statement about that. Please keep that for a separate PR.